.evergreen/provision-atlas.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+. .evergreen/utils.sh
+
+PYTHON_BINARY=$(find_python3)
+
+# Should be called from src
+EVERGREEN_PATH=$(pwd)/.evergreen
+TARGET_DIR=$(pwd)/$DIR
+PING_ATLAS=$EVERGREEN_PATH/ping_atlas.py
+SCAFFOLD_SCRIPT=$EVERGREEN_PATH/scaffold_atlas.py
+DEPLOYMENT_NAME=$DIR
+
+# Download the mongodb tar and extract the binary into the atlas directory
+set -ex
+curl https://fastdl.mongodb.org/mongocli/mongodb-atlas-cli_1.14.0_linux_x86_64.tar.gz -o atlas.tgz
+tar zxf atlas.tgz
+mv mongodb-atlas-cli_1.14.0* atlas
+
+# Create a local atlas deployment and store the connection string as an env var
+$atlas deployments setup $DIR --type local --force
+$atlas deployments start $DIR
+CONN_STRING=$($atlas deployments connect $DIR --connectWith connectionString)
+
+# Make the atlas directory hold the virtualenv for provisioning
+cd atlas
+
+$PYTHON_BINARY -m venv .
```

Comment:
Is this activated?

Reply:
Good point. As additional insurance, I'll call `source ./bin/activate`